index.bs

```diff
@@ -12521,6 +12518,19 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="preventextensions">Platform object \[[PreventExtensions]] method</h4>
+
+The internal \[[PreventExtensions]] method of every [=platform object=] |O|
+that implements an [=interface=] which
+[=support indexed properties|supports indexed=] or
+[=support named properties|named properties=]
+must behave as follows when called:
+
+<ol class="algorithm">
```

Comment:
I'm removing those single step algorithms as part of #198 but keeping it around here for consistency.